rust/connlib/tunnel/src/client.rs

```diff
@@ -89,6 +89,8 @@ where
     }
 
     pub fn remove_resource(&mut self, id: ResourceId) {
+        tracing::info!(%id, "Removing resource");
```

Comment:
```suggestion
        tracing::debug!(%id, "Removing resource");
```

I might suggest debug level here -- this will happen anytime a policy, group membership, resource, or site affects the resource available to a client.

Reply:
As a policy, whenever we print an ID we should, save there is a good reason not to, retrieve the actual object and print that. It'll be way more useful to debug.

Reply:
I think that will be very noisy and make the logs super hard to read in 90% of the cases. Resource, gateway and client IDs should be stable so we can always look them up in the portal, right?